### ISSUE TITLE
[BLUS30804] Syndicate - Blur Effect Missing / Flashing

### ISSUE THREAD

**Issue by Gazingpenguin** [RPCS3/rpcs3#13811]
### Quick summary

Blur effect seems to be broken. Also in sections with light-shafts, they will strobe.

### Details

[BLUS30804] Syndicate use to require WCB + Force CPU blit to start and the game had immense graphical issues.  I suspect that [[Merged] PR #13799 by kd-11](https://github.com/RPCS3/rpcs3/pull/13799) may have solved it, but I am unsure. Thankfully, now it is in a semi-playable state with only visual glitches.

Firstly, Blur effect seems to be broken. Below are screen captures from PS3 gameplay beside the current RPCS3 with default settings. 
![a92f3c81351e5ecce71b8daf2bcf4b6b](https://user-images.githubusercontent.com/131831461/236601681-1f849114-ecb6-412a-aaae-4c2f715df00f.png)
![9d41f1626b9e8cc04bb79d511f23f175](https://user-images.githubusercontent.com/131831461/236601821-43cf4e9e-72cd-4ee2-8333-1ed8071cc343.jpg)

![c4bc3ff21c404efd2f82382108c7bf07](https://user-images.githubusercontent.com/131831461/236601807-7d6efcc9-d1c1-4ebe-9eed-737b92aa7f43.jpg
![b0765502d20699fc32f71c2e41ede770](https://user-images.githubusercontent.com/131831461/236601841-567e2fc3-9ee1-4ba5-bdbf-554a2fcf0214.jpg)
)

### Attach a log file

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/11411408/RPCS3.log.gz)

### Attach capture files for visual issues

https://www.dropbox.com/s/4zxytm9m3noa5my/captures.zip?dl=0

### System configuration

OS: Windows, Major: 10, Minor: 0, Build: 22000, Service Pack: none, Compatibility mode: 0
Processor: 11th Gen Intel(R) Core(TM) i7-11700F @ 2.50GHz | 16 Threads | 15.87 GiB RAM | TSC: 2.496GHz | AVX-512+ | FMA3
Ram: 16 GBs
GPU:  NVIDIA GeForce RTX 3070' running on driver 531.61.0.0
SYS: Firmware version: 4.89

### Other details

The game has trouble starting as well, Setting Shader to Auto seems to have worked but this is untested.

**Comment by digant73** [comment]
is that issue still present?

**Comment by Triticum0** [comment]
Yes, at least bloom issue is https://www.youtube.com/live/rtyZVyDRuZk?t=336&si=h3ocCF97ei-loW12

**Comment by zhuyijun1573** [comment]
can be playedble now ?

**Comment by digant73** [comment]
please an update on current build
